### DISCUSSION TITLE
Header still partially showing on iOS

### DISCUSSION THREAD

**Discussion by TimBeekman** [NemesisRE/kiosk-mode#205] [Q&A] [answered]
The header is hidden but i still see a blue bar at the top:

![Previeuw](https://github.com/NemesisRE/kiosk-mode/assets/89469564/db066584-ce5b-421f-8323-468eeea06006)

**Comment by NemesisRE** [comment]
Only on mobile or also in a desktop browser?

**Reply by TimBeekman** [reply]
Only on mobile

**Reply by elchininet** [reply]
Hi @TimBeekman,
Is it visible on the screenshot?
It could be caused by my screen but could yo point out in which side exactly it is shown?
Regards

**Reply by TimBeekman** [reply]
Its the blue bar at the top, where the time and cellular data is shown

**Reply by elchininet** [reply, marked as answer]
Hi @TimBeekman,

I think that you are referring to the native iOS header of the telephone. That is out of what `kiosk-mode` can manipulate. For example, this is my Android device:

![Screenshot_2024-02-28-15-05-55-55_c3a231c25ed346e59462e84656a70e50.jpg](https://github.com/NemesisRE/kiosk-mode/assets/3728220/cbebec4d-5333-4745-ad2f-30f064a773cb)

And after hiding the header:

![Screenshot_2024-02-28-15-06-59-85_c3a231c25ed346e59462e84656a70e50.jpg](https://github.com/NemesisRE/kiosk-mode/assets/3728220/0abe8ee0-a46c-4341-a98b-17baf36eea96)

What you are searching for, at least on the Android app (on iOS the settings are different) is located in:

Settings > Companion App > Full Screen

Regards

**Reply by elchininet** [reply]
Follow this thread:

https://community.home-assistant.io/t/home-assistant-companion-app-to-do-full-screen-and-camera-like-kiosks/376576/7

And this Github discussion:

https://github.com/home-assistant/iOS/discussions/1720#discussioncomment-3587931

**Reply by TimBeekman** [reply]
I do believe you're right about kiosk mode not having any influence on this.

however, i hope you can help me. If I enable full screen mode in the app-settings, the time disappears, but the blue bar remains. If i make a custom theme to hide the bar, i lose the possibility to automatically change theme color based on dark or light mode enabled on the phone. Do you have any further advice?

**Reply by elchininet** [reply]
I really don't know what else you can do that is not provided natively from the App. That bar belongs to the system, it is a native bar and as I understand it can be controlled from apps. I think you should raise your question in the repo of the iOS app.
